### PR TITLE
[ADDED UPDATE STUDENT FUNCTIONALITY]

### PR DESCRIPTION
**Pull request by RomeoPetryshyn**
Added update student functionality in the same modal window, used http request to adminUser in order to get username and email which are not being returned by http request to student entity. Set up needed validations for edit and add forms, fixed some minor bugs, changed design as needed for demo and also added some minor design features. Tried to build project for production - building and running completely fine using live server

### CODE REVIEW

File: src/app/admin/students/services/students.service.ts [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+  createStudent(body): Observable<CreareUpdateStudentsInterface|ResponseInterface> {
+    return this.httpClient.post<CreareUpdateStudentsInterface|ResponseInterface>(`${this.createStudentUrl}`, body);
+  }
+
+  updateStudent(id: number, body): Observable<CreareUpdateStudentsInterface|ResponseInterface> {
+    return this.httpClient.post<CreareUpdateStudentsInterface|ResponseInterface>(`${this.updateStudentUrl}/${id}`, body);
+  }
+
+  getUserInfo(id: string){
+    return this.httpClient.get(`${this.getUserInfoURL}/${id}`);
```
Comment (kravtsivkostya) [inline src/app/admin/students/services/students.service.ts:41]: Why do u need this lines of code if we already have same logic in shared service?

File: src/app/admin/students/services/students.service.ts [hunk 2]
```diff
@@ -1,14 +1,45 @@
...
 
 @Injectable()
 export class StudentsService {
 
+  private createStudentUrl = 'Student/insertData';
+  private readStudentUrl = 'Student/getStudentsByGroup';
+  private updateStudentUrl = 'Student/update';
+  private deleteStudentUrl = 'Student/del';
+  private getUserInfoURL = 'AdminUser/getRecords';
+
```
Comment (kravtsivkostya) [inline src/app/admin/students/services/students.service.ts:16]: Same unnecessary code

File: src/app/admin/students/services/students.service.ts (hunk 1 above)
Comment (RomeoPetryshyn) [reply src/app/admin/students/services/students.service.ts:41]: Got it, will be refactored with using shared logic and tested everything in the next pull request today after merge of this one

File: src/app/admin/students/services/students.service.ts (hunk 2 above)
Comment (RomeoPetryshyn) [reply src/app/admin/students/services/students.service.ts:16]: Got it)

**Review by kravtsivkostya — APPROVED** [review]

**Pull request merged**